testcases/kernel/containers/pidns/pidns13.c

```diff
@@ -192,12 +192,12 @@ int main(int argc, char *argv[])
 	}
 
 	/* Create container 1 */
-	*cinit_no = 1;
-	cpid1 = ltp_clone_quick(CLONE_NEWPID | SIGCHLD, child_fn, cinit_no);
+	cinit_no[0] = 1;
+	cpid1 = ltp_clone_quick(CLONE_NEWPID | SIGCHLD, child_fn, &cinit_no[0]);
 
 	/* Create container 2 */
-	*cinit_no = 2;
-	cpid2 = ltp_clone_quick(CLONE_NEWPID | SIGCHLD, child_fn, cinit_no);
+	cinit_no[1] = 2;
+	cpid2 = ltp_clone_quick(CLONE_NEWPID | SIGCHLD, child_fn, &cinit_no[1]);
 	if (cpid1 < 0 || cpid2 < 0) {
 		tst_resm(TBROK, "parent: clone() failed.");
 	}
```

Comment:
Looks correct, you cannot assume any ordering on cloned() processes.

However I do not understand why do we allocate the memory instead of just passing the value directly in the pointer, i.e. (void*)(intptr_t)1, which would fix the problem as well.

Also the patch misses signed-of-by: line.

And lastly but not least sorry for neglecting the patch for a long time.